### ISSUE TITLE
NameError: name 'use_gemm_woq_plugin' is not defined

### ISSUE THREAD

**Issue by wjueyao** [triton-inference-server/tensorrtllm_backend#211]
When I run build.py using tensorrtllm_backend built from main branch
```
# Build model with both INT8 weight-only and INT8 KV cache enabled
python build.py --ft_model_dir=/weights/trt_llm/2023.12.12/llama_7B_int8_kv_cache/4-gpu/ \
                --dtype float16 \
                --remove_input_padding \
                --use_gpt_attention_plugin float16 \
                --paged_kv_cache \
                --use_inflight_batching \
                --enable_context_fmha \
                --use_gemm_plugin float16 \
                --max_batch_size 64  \
                --vocab_size 32016  \
                --output_dir /weights/trt_llm/2023.12.12/convert/4-gpu \
                --int8_kv_cache \
                --use_weight_only \
                --world_size 4 \
                --tp_size 4
```

I got the following error

```
Traceback (most recent call last):
  File "/tensorrtllm_backend/tensorrt_llm/examples/llama/build.py", line 839, in <module>
    build(0, args)
  File "/tensorrtllm_backend/tensorrt_llm/examples/llama/build.py", line 783, in build
    engine = build_rank_engine(builder, builder_config, engine_name,
  File "/tensorrtllm_backend/tensorrt_llm/examples/llama/build.py", line 641, in build_rank_engine
    load_from_binary(tensorrt_llm_llama,
  File "/tensorrtllm_backend/tensorrt_llm/examples/llama/weight.py", line 917, in load_from_binary
    if not use_gemm_woq_plugin:
NameError: name 'use_gemm_woq_plugin' is not defined
```

**Comment by nv-samcheng** [comment]
I'll take a look.

**Comment by nv-samcheng** [comment]
@wjueyao Thanks for reporting this issue.

We are fixing this issue.

Here is WAR for this issue if you would unblock it.
Edit this [file](https://github.com/NVIDIA/TensorRT-LLM/blob/f7eca56161d496cbd28e8e7689dbd90003594bd2/examples/llama/weight.py#L761)

 ```
def load_from_binary(tensorrt_llm_llama: LLaMAForCausalLM,
  dir_path,
  mapping=Mapping(),
  fp16=False,
  multi_query_mode=False):
```

to

```
def load_from_binary(tensorrt_llm_llama: LLaMAForCausalLM,
  dir_path,
  mapping=Mapping(),
  fp16=False,
  multi_query_mode=False,
  use_gemm_woq_plugin=True)
```

**Comment by wjueyao** [comment]
@nv-samcheng Thanks. It working now. However, the engines I build is generating weird results. I will open up a new issue to follow this problem

**Comment by dominicshanshan** [comment]
> @wjueyao Thanks for reporting this issue.
> 
> We are fixing this issue.
> 
> Here is WAR for this issue if you would unblock it. Edit [examples/llama/weight.py](https://github.com/NVIDIA/TensorRT-LLM/blob/f7eca56161d496cbd28e8e7689dbd90003594bd2/examples/llama/weight.py#L761)
> 
> ```python
> def load_from_binary(tensorrt_llm_llama: LLaMAForCausalLM,
>     dir_path,
>     mapping=Mapping(),
>     fp16=False,
>     multi_query_mode=False):
> ```
> 
> to
> 
> ```python
> def load_from_binary(tensorrt_llm_llama: LLaMAForCausalLM,
>     dir_path,
>     mapping=Mapping(),
>     fp16=False,
>     multi_query_mode=False,
>     use_gemm_woq_plugin=True)
> ```

can we merge this fix into main branch? still not fix yet on v0.7.1

**Comment by nv-samcheng** [comment]
@dominicshanshan This issue is fixed in main branch.
Please target on latest release.
@byshiue please help me to close this bug, Thanks.

**Comment by dominicshanshan** [comment]
> @dominicshanshan This issue is fixed in main branch. Please target on latest release. @byshiue please help me to close this bug, Thanks.

@nv-samcheng, could you try v0.7.1? I still face this error in v0.7.1, I use follow command to get v0.7.1 from GitHub:

`git clone -b v0.7.1 --single-branch https://github.com/NVIDIA/TensorRT-LLM.git`

am I wrongly pulled from GitHub?

**Comment by byshiue** [comment]
@dominicshanshan  nv-samcheng mentioned the issue is fixed in latest `main` branch instread of v0.7.1. Please take a try. Close this issue, but feel free to ask question if you still encounter any issue. We will reopen it if needed.
